microsoft/testsuites/performance/common.py

```diff
@@ -393,6 +394,9 @@ def perf_iperf(
                     )
                 )
                 current_server_port += 1
+                time.sleep(0.5)
+            # add sleep to avoid server is not ready when client connects to it.
+            time.sleep(5)
```

Comment:
It can be fixed by use `wait_output` on process. 5 seconds is too long, and hard sleep isn't stable.

Reply:
the log is redirected into json file till the that iteration test done. wait_output doesn't work here.

Reply:
Is it possible to check the json file? BTW, we should use small PR to make other changes effective early.

Reply:
the json file will generate in the last, I will remove this commit from this PR.